test/helper.rb

```diff
@@ -7,10 +7,18 @@
 I18n.load_path << File.expand_path("../locale/en.yml", __FILE__)
 
 class User < Minidoc
+  self.collection_name = "users"
```

Comment:
@wfleming no - just wanted to be explicit so it's clear in the code why we're using two classes.